### ISSUE TITLE
Intellij run error: scalac: No 'scala-library*.jar' in Scala compiler classpath in Scala SDK 

### ISSUE THREAD

**Issue by bbarker** [lihaoyi/mill#331]
I get this error: `Error:scalac: No 'scala-library*.jar' in Scala compiler classpath in Scala SDK scala-library-2.12.6-sources.jar` when trying to run a test (utest) in IntelliJ. I have added what seems to be the correct scala libraries under project structure, and restarted IntelliJ:

![image](https://user-images.githubusercontent.com/916366/40182440-642fa736-59b9-11e8-868f-a368121d4517.png)

**Comment by raskam** [comment]
This happened to me only the first time I generated Idea project with Mill. I fixed it by setting Java SDK and Scala SDK manually and since then GenIdea works as expected. It works properly even when I remove `.idea/`, `.idea_modules/` and `out/` directories, close Idea, regenrate Idea project and open it in Idea again. So I suspect that is is related to Idea caches outside the project root.

I'm using Idea community 2018.1 with Scala 2018.1.10 on macOS, just for the reference.
